build.gradle

```diff
@@ -17,6 +17,17 @@ buildscript {
     }
 }
 
+plugins {
```

Comment:
According to this, this syntax is still incubating in Gradle. See if you can't fix it to use the apply syntax